### PR TITLE
Add debug.enable.vga support

### PR DESCRIPTION
**Pull request by mikem-zed**
Sometimes it is required to debug EVE issues when HW is accessible directly e.g. in the field but access to the cloud is problematic e.g. due to bad connectivity.
    
Another case is EVE onboarding problems when SSH is not yet available.
    
VGA console may help in both cases. We only return VGA devices that were marked as boot devices. Console output won't be visible on others anyway. It also allows us to debug issues using VGA console while other GPUs are still  assigned to aplications

### CODE REVIEW

**Review by rvs — APPROVED** [review]
LGTM

File: pkg/pillar/cmd/zedagent/parseconfig.go [hunk 1]
```diff
@@ -2159,6 +2159,7 @@ func parseConfigItems(config *zconfig.EdgeDevConfig, ctx *getconfigContext) {
 	// That way bringup of new hardware models can be done using an
```
Comment (eriknordmark) [inline pkg/pillar/cmd/zedagent/parseconfig.go:2159]: Makes sense to update this comment to also mention VgaAccess on L2154 and screen on L2160

**Review by eriknordmark — APPROVED** [review]
LGTM. One comment update which would make sense to do.

**Review by rvs — APPROVED** [review]
LGTM

**Pull request merged**